File: src/test/java/com/dtforce/migen/test/mock2/spring/Mock2.java
package com.dtforce.migen.test.mock2.spring;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;

import com.dtforce.migen.intergrations.spring.EnableMigrationGenerator;

@SpringBootApplication
@EntityScan({"com.dtforce.migen.test.mock2"})
@EnableMigrationGenerator
public class Mock2
{
    public static void main(String[] args) {
		SpringApplication.run(Mock2.class);
	}
}

File: src/test/kotlin/com/dtforce/migen/test/mock3/spring/Mock2.kt
package com.dtforce.migen.test.mock3.spring

import com.dtforce.migen.intergrations.spring.EnableMigrationGenerator
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(Mock3Configuration::class)
@EntityScan("com.dtforce.migen.test.mock3")
@EnableMigrationGenerator
open class Mock3 {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(Mock3::class.java)
        }
    }
}